gaffer-worker/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2016-2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+jwt.secret=javainuse
```

Comment:
What is this used for? I'm just a little confused because this is hardcoded elsewhere.

Reply:
We need this otherwise JWT does not work

Reply:
It's still a todo/work-in-progress, looking to implement a better solution in this ticket https://github.com/gchq/gaffer-experimental/issues/32 in another branch. I guess it currently works for now as a 'dev mode 'whilst gaffer-experimental is still in this experimental development phase, if that's ok?

Can implement a better solution in to this branch also?